packages/components/src/internal/components/picklist/ChoosePicklistModal.tsx

```diff
@@ -500,7 +500,7 @@ export const ChoosePicklistModal: FC<ChoosePicklistModalProps> = memo(props => {
                         setError(resolveErrorMessage(e) ?? 'Failed to retrieve picklist selection.');
                     }
                 } else if (selections) {
-                    ids_ = [...selections];
+                    ids_ = Array.from(selections).map(s => parseInt(s, 10));
```

Comment:
can this use `queryModel.getSelectedIdsAsInts()` instead of `selections`? looks like that is doing this same conversion.